app/src/main/res/values/strings.xml

```diff
@@ -306,6 +306,8 @@ The data saved on your smartphone and the connections to the server are encrypte
 Above the line, describe your queries or issues with the app. If possible, do not delete the text below. It will help the Immuni Support team to better assist you. 
 Operating system: iOS 13.5.1; Model: iPhone XS; Exposure notifications: Active; [other useful info, same as the info displayed on the contact support screen]</string>
   <string name="settings_setting_support">Contact Support</string>
+  <string name="settings_setting_share">Condividi l\'app</string>
+  <string name="settings_setting_share_message">Aiutaci a combattere insieme l’epidemia di COVID-19. Scarica anche te Immuni. https://www.immuni.italia.it/</string>
```

Comment:
> Scarica anche te Immuni

"Anche tu". "*Anche te*" is a regionalism.

Reply:
Thanks for the input, fixed.